docs/reference/how-to/fix-common-cluster-issues.asciidoc

```diff
@@ -100,6 +100,128 @@ POST _cache/clear?fielddata=true
 ----
 // TEST[s/^/PUT my-index\n/]
 
+**Configure the script compilation circuit breaker**
+
+If you frequently trigger the <<script-compilation-circuit-breaker,script
```

Comment:
While this may be useful for a limited set of 7.x versions, I believe we still plan to remove the need for this setting, see https://github.com/elastic/elasticsearch/issues/62899 (@stu-elastic can you confirm?). So I wonder if we should avoid suggesting using the context specific setting, which will go away by 8.0.

Reply:
Thanks @rjernst.

@stu-elastic Let me know if you have any recommendation on a replacement setting here.

AFAIK `script.max_compilations_rate` was deprecated in 7.9 and would required a rolling restart if `script.disable_max_compilations_rate` isn't set to `false` (it's static and `true` by default). It looks like `script.max_compilations_rate` is currently removed in 8.0.

If this is the best solution for now, I can publish these docs and update them after a PR for #62899 is merged.

Reply:
After giving this some more thought, I'm going to remove the script compilation circuit breaker tip entirely.

The script circuit breaker is pretty different from our other circuit breakers so it should probably get it's own section. We should probably also expand this to include an example error message and talk about increasing the script cache.